src/libraries/System.Private.CoreLib/src/System/Runtime/InteropServices/WasmImportLinkageAttribute.cs

```diff
@@ -0,0 +1,17 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace System.Runtime.InteropServices
+{
+    /// <summary>
+    /// Used to control WASM import module linkage for an associated P/Invoke.
+    /// </summary>
```

Comment:
@SingleAccretion or @yowl Would either of you be interested in providing some addtional documentation for this attribute? I still don't feel like I understand the nuances of the WASM scenarios to speak intelligently about it.

Reply:
```
Specifies that the P/Invoke marked with this attribute should be linked in as a WASM import.
```
Should be sufficient I think.

Are we allowed links in docs? If so, it could link directly to the spec as well:
```
https://webassembly.github.io/spec/core/syntax/modules.html#imports.
```